src/store/reducers/Chat.ts

```diff
@@ -0,0 +1,34 @@
+import { ChatActions, ChatActionTypes } from "store/actions/Chat";
+import { CHAT_TYPES } from "types/Chat";
+
+type PrivateChatState = {
+  openedChatType: CHAT_TYPES.PRIVATE_CHAT;
+  recipientId: string | null;
```

Comment:
What does `null` represent here? As a general rule, we basically never want to be using `null`, or at the very least, there needs to be a very compelling case as to why we would use it. What does it offer us here that `undefined` doesn't?